Documentation/disaster-recovery.md

```diff
@@ -106,3 +106,107 @@ kubectl create -f operator.yaml
 ```
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `monCount`.
+
+# Adopt an existing Rook Ceph cluster into a new Kubernetes cluster
+
+## Situations this section can help resolve
+
+1. The Kubernetes environment underlying a running Rook Ceph cluster failed catastrophically, requiring a new Kubernetes environment in which the user wishes to recover the previous Rook Ceph cluster.
+2. The user wishes to migrate their existing Rook Ceph cluster to a new Kubernetes environment, and downtime can be tolerated.
+
+### Prerequisites
+
+1. A working Kubernetes cluster to which we will migrate the previous Rook Ceph cluster.
+2. At least one Ceph mon db is in quorum, and sufficient number of Ceph OSD is `up` and `in` before disaster.
+3. The previous Rook Ceph cluster is not running.
+
+Tested in `rook/ceph:v1.0.3` and `ceph/ceph:v14.2.1-20190430`.
+
+### Overview for Steps below
+1. Start a new and clean rook cluster, with old `CephCluster` `CephBlockPool` `CephFilesystem` `CephNFS` `CephObjectStore`.
+2. Shut the new cluster down when it has been created successfully.
+3. Replace ceph-mon data with old one.
+4. Fix `fsid` in `secrets/rook-ceph-mon`.
```

Comment:
Nit: I would prefer be more specific here than "Fix." Instead maybe "Replace `<field>` in `<blah>` with that of the old cluster"

Reply:
fixed